apps/chameleon/components/ui/card.tsx

```diff
@@ -9,7 +9,7 @@ const Card = React.forwardRef<
   <div
     ref={ref}
     className={cn(
-      'rounded-lg border bg-card text-card-foreground shadow-sm',
+      'h-full w-full border bg-card text-card-foreground shadow-sm',
```

Comment:
dont edit the atom, if you need to extend this card into a unique new card, make a molecule that has the extensions that you need.